x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/huggingface/HuggingFaceServiceSettings.java

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Exactly the same as `HuggingFaceElserServiceSettings` but I figured I'd create a new one with the right named writeable name in case we evolve it by adding more fields later.